tests/unit/h/emails/reply_notification_test.py

```diff
@@ -89,20 +89,6 @@ def test_falls_back_to_individual_page_if_no_bouncer(
         html_renderer.assert_(**expected_context)  # noqa: PT009
         text_renderer.assert_(**expected_context)  # noqa: PT009
 
-    def test_supports_non_ascii_display_names(
-        self,
-        notification,
-        pyramid_request,
-        parent_user,
-        reply_user,
-    ):
-        parent_user.display_name = "Parent 👩"
-        reply_user.display_name = "Child 👧"
-
-        email = generate(pyramid_request, notification)
-
-        assert email.subject == "Child 👧 has replied to your annotation"
```

Comment:
The only other place where these values are set now is the email body, but since the renderer is mocked, we get the body as an empty string.

Because of that I removed this test, as it would require rendering the email with the contextual parameters "for real", but I'm happy to keep it if there's some way to continue testing this.